data/store/mongo/mongo_test.go

```diff
@@ -94,11 +94,9 @@ var _ = Describe("Mongo", func() {
 		var logger log.Logger
 		var mongoConfig *mongo.Config
 		var mongoStore *mongo.Store
-		var err error
```

Comment:
Not really an issue. Just getting in the habit of declaring variables as close to their use as possible. In this case, the err variable is essentially local to each specific test function so should be declared there. The other variables are used in the `BeforeEach`  as well as each specific test function so need to be declared in the outer scope.